modules/ossm-installing-gateway-using-gateway-injection.adoc

```diff
@@ -0,0 +1,246 @@
+:_mod-docs-content-type: Concept
+[id="ossm-about-gateway-injection_{context}"]
+= Installing a gateway by using gateway injection
+:context: ossm-installing-gateway-using-gateway-injection
+
+This procedure explains how to install a gateway by using gateway injection.
+
+.Prerequisites
+
+* You have installed the {SMProduct} Operator.
```

Comment:
Should we specify that it has to be the OpenShift Service Mesh 3 Operator?

```suggestion
* You have installed the {SMProduct} 3 Operator.
```

Reply:
done